src/openforms/authentication/contrib/digid_oidc/tests/test_admin.py

```diff
@@ -0,0 +1,142 @@
+from unittest.mock import patch
+
+from django.test import override_settings
+from django.urls import reverse
+
+from django_webtest import WebTest
+
+from openforms.accounts.tests.factories import SuperUserFactory
+from openforms.config.models import GlobalConfiguration
+from openforms.forms.models import Form
+from openforms.forms.tests.factories import FormFactory
+
+from ..models import OpenIDConnectPublicConfig
+
+
+@override_settings(CORS_ALLOW_ALL_ORIGINS=True, IS_HTTPS=True)
+class DigiDOIDCFormAdminTests(WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        config = OpenIDConnectPublicConfig.get_solo()
+        config.enabled = True
```

Comment:
we prefer the `@patch("...SoloModel.get_solo", return_value=SoloModel(...))` pattern to isolate tests better (as cache is skipped that way) and it's slightly more performant -> faster tests

for class wide patches, you can do `patcher = patch(...)` in the `setUp()` method and call `patcher.start()` etc - you can find samples of that throughout the codebase